Graphics/Blank/JavaScript.hs

```diff
@@ -624,7 +618,17 @@ jsEscapeChar '\t' = "\\t"
 jsEscapeChar '\v' = "\\v"
```

Comment:
It's a bit curious that this version of the function has a special case for `\v`, but the one in GHCJS does not.

Reply:
The version in GHJS would treat this as a control symbol and render as `\x0B`. I thought your version is more succinct and both are supported by JS, as far as I can tell. So I left yours.

Reply:
Gotcha, it sounds like either one will work. [This site](https://mathiasbynens.be/notes/javascript-escapes) indicates that using `\x0B`  is important if you care about compatibility with Internet Explorer < 9, but if you're using that old of an IE version in this day and age, you have bigger problems to worry about :)